patches/minecraft/net/minecraft/potion/Potion.java.patch

```diff
@@ -88,6 +88,17 @@
 +        ret.add(new net.minecraft.item.ItemStack(net.minecraft.init.Items.field_151117_aB));
 +        return ret;
 +    }
++    
++    /**
++     * Used as a last resort for determining {@code PotionEffect} sort order in GUIs.
```

Comment:
No need for the intensity here, it can just be
`Used for determining {@code PotionEffect} sort order in GUIs.`